core/lib/generators/solidus/install/install_generator.rb

```diff
@@ -61,25 +58,15 @@ def prepare_options
       @run_migrations = options[:migrate]
       @load_seed_data = options[:seed]
       @load_sample_data = options[:sample]
+      @selected_frontend = detect_frontend_to_install
+      @selected_authentication = detect_authentication_to_install
 
       # Silence verbose output (e.g. Rails migrations will rely on this environment variable)
       ENV['VERBOSE'] = 'false'
 
       # No reason to check for their presence if we're about to install them
       ENV['SOLIDUS_SKIP_MIGRATIONS_CHECK'] = 'true'
 
-      if options[:with_authentication] != nil
-        warn \
```

Comment:
If we're removing the warning, can't we remove the option altogether now?

Reply:
The warning is still there, just moved it to `detect_authentication_to_install`.